packaging/ubuntu-14.04/snapd.install

```diff
@@ -11,11 +11,14 @@ etc/profile.d
 etc/X11
 # bash completion
 data/completion/snap /usr/share/bash-completion/completions
+data/completion/complete.sh /usr/lib/snapd/
+data/completion/etelpmoc.sh /usr/lib/snapd/
 # udev, must be installed before 80-udisks
 data/udev/rules.d/66-snapd-autoimport.rules /lib/udev/rules.d
+# snap/snapd version information
 data/info /usr/lib/snapd/
 
-# snap-confine
```

Comment:
these seemingly random changes are so that the diff between the 14.04 and 16.04 snapd.installs are only about what's important